charts/sonarqube-dce/templates/jdbc-config.yaml

```diff
@@ -1,3 +1,5 @@
+{{ $applicationNodes := fromYaml (include "applicationNodes" . ) }}
```

Comment:
i asked this question to myself, and I preferred adding it everywhere so in case someone add a `.Values.ApplicationNodes` in one of those file during a feature/hardening/whatever it would still work correctly. Kind of as a safeguard. let me know what you think.